README.md

```diff
@@ -82,6 +82,45 @@ If Heroku idles your application, measurements will not be sent until it receive
 
 Please note that a Paid Plan is required for tracking Custom Metrics in Heroku. Choose a Librato add-on plan here (https://elements.heroku.com/addons/librato#pricing).
 
+##### Metric Suites
+
+The metrics recorded by `librato-rails` are organized into named metric suites that can be selecitvely enabled/disabled:
+
+* `rack`: The `rack.request.total`, `rack.request.time`, `rack.request.slow`, and `rack.request.queue.time` metrics
+* `rack_status`: All of the `rack.request.status.*` metrics
+* `rack_method`: All of the `rack.request.method.*` metrics
+* `rails_cache`: All of the `rails.cache.*` metrics
```

Comment:
note there are also `rails.request.status.*` metrics and `rails.request.method.*` metrics, so probably need `rails_status` and `rails_method` here?